walkthroughs/howto-mutual-tls-acm/README.md

```diff
@@ -0,0 +1,277 @@
+# Configuring Mutual TLS with ACM PCA TLS Certificates
+
+In this walkthrough, like the [basic file-based TLS example](../howto-tls-file-provided), we'll enable TLS encryption with mutual (two-way) authentication between two endpoints in App Mesh using X.509 certificates.
+
+## Introduction
+
+In App Mesh, traffic encryption is originated and terminated by the Envoy proxy. This means your application code is not responsible for negotiating a TLS-encrypted connection, instead allowing the local proxy to negotiate and terminate TLS on your application's behalf.
+
+In a basic TLS encryption scenario (for example, when your browser originates an HTTPS connection), the server would present a certificate to any client. In Mutual TLS, both the client and the server present a certificate to each other, and both validate the peer's certificate.
+
+Validation typically involves checking at least that the certificate is signed by a trusted Certificate Authority, and that the certificate is still within its validity period. The application (in our case, Envoy) may also choose to validate other aspects of the certificate, such as the Subject Alternative Name identity signed into the certificate.
+
+In this guide, we will be configuring Envoy proxies using certificates hosted in AWS Secrets Manager, which a modified Envoy image will retrieve during startup. We will have a virtual gateway connected to a single backend service. Both the gateway and backend proxies will present certificates signed by the same Certificate Authority (CA), though you could choose to use separate CAs.
```

Comment:
Can we also mention ACM and the lambda that's used to populate secrets manager here? When does the lambda get called?

Reply:
Lambda gets called after ACM is created as a custom resource in the template file
https://github.com/aws/aws-app-mesh-examples/blob/54acdd1b3d8e3d89efc4e530b9527226f278cd5d/walkthroughs/howto-mutual-tls-acm/infrastructure/deploy.yaml#L524-L605

Reply:
I have add this in the ACM template and mentioned in the README